src/main/java/webs/hillel/OrderController.java

```diff
@@ -6,31 +6,31 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/orders")
+@RequestMapping("/api/orders")
 public class OrderController {
     private OrderRepository orderRepository;
 
     public OrderController(OrderRepository orderRepository) {
         this.orderRepository = orderRepository;
     }
 
-    @PostMapping("/")
+    @PostMapping
     public Order addOrder(@RequestBody Order order) {
         return orderRepository.addOrder(order);
     }
 
     @GetMapping("/{id}")
     public ResponseEntity<Order> getOrderById(@PathVariable int id) {
-        Order order = orderRepository.getOrderById(id);
-        if (order != null) {
+        try {
```

Comment:
You added try-catch here, why? Did you read my previous comment about this endpoint? This is extremely important!
_The best practice for writing controllers is making them as simple as possible and **having no try-catch, if-else, etc**. **All the logic should be encapsulated in the service layer. If an order wasn't found, then throw an exception inside the service**._ Having this in mind would be great if you refactor this if-else to eliminate it.